CHANGELOG.md

```diff
@@ -79,6 +79,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 - Standalone website:
   - Correctly display creation form when there is no playlist
+- Blacklist the refresh token on the frontend side (#2265)
```

Comment:
Not in the good section

Reply:
https://github.com/openfun/marsha/pull/2265/commits/e5200f0dcbb9eb981cdd71403b07cd768e5d97f9